torch/_inductor/codegen/cpp_template_kernel.py

```diff
@@ -324,7 +324,12 @@ def __init__(
         input_nodes: List[ir.Buffer],
         layout: ir.Layout,
         make_kernel_render: Callable[
-            [ir.CppTemplateBuffer, Optional[List[ir.IRNode]]], str
+            [
+                ir.CppTemplateBuffer,
+                bool,
```

Comment:
optional or not?

Reply:
Optional in the `render` function but not optional here